src/bank.rs

```diff
@@ -1256,14 +1306,19 @@ mod tests {
 
     #[test]
     fn test_bank_new() {
-        let mint = Mint::new(10_000);
+        let dummy_leader_id = Keypair::new().pubkey();
+        let dummy_leader_tokens = 1;
+        let mint = Mint::new(10_000, dummy_leader_id, dummy_leader_tokens);
```

Comment:
Can you leave `Mint::new()` unmodified and instead added a `new_with_leader` constructor?

Reply:
Done